src/s3/activation.ts

```diff
@@ -34,7 +33,7 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
         vscode.commands.registerCommand('aws.s3.uploadFile', async (node: S3BucketNode | S3FolderNode) => {
             if (!node) {
                 const awsContext = new DefaultAwsContext(ext.context)
```

Comment:
This doesn't work. AwsContext is a global that is initialized in extension.ts. Creating a new one won't have the state we need from it.